<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addItemView</class>
 <widget class="QWidget" name="addItemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="iDlabel">
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="iDEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelbtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color:  rgb(13, 105, 239);
    border-radius: 3px;
	color: white;
}
QPushButton:pressed {
    background-color: rgb(203, 218, 249);
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserimento nuovo Oggetto</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioNotebook">
      <property name="text">
       <string>Notebook</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioSmartphone">
      <property name="text">
       <string>Smartphone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioSmartwatch">
      <property name="text">
       <string>Smartwatch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirmbtn">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>410</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color:  rgb(13, 105, 239);
    border-radius: 3px;
	color: white;
}
QPushButton:pressed {
    background-color: rgb(203, 218, 249);
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Inserisci</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="branlabel">
      <property name="text">
       <string>Brand</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="brandEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="processorlabel">
      <property name="text">
       <string>Processor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="processorEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="modellabel">
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="modelEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="displaylabel">
      <property name="text">
       <string>Display(&quot;)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="displayEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="pricelabel">
      <property name="text">
       <string>Price($)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="priceEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="memorylabel">
      <property name="text">
       <string>Memory(Gb)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="memoryEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="GPUlabel">
      <property name="text">
       <string>GPU</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="GPUEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="cam1label">
      <property name="text">
       <string>Cam1(Megapx)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cam1Edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_11">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="cam2label">
      <property name="text">
       <string>Cam2(Megapx)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cam2Edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_fanles">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fanless</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_backL">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Backlight</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_finger">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fingerprint</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="Userlabel">
      <property name="text">
       <string>User</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="UserBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_13">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="certificatelabel">
      <property name="text">
       <string>Certificate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="certificateEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_14">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QLabel" name="materiallabel">
      <property name="text">
       <string>MaterialC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="materialEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_15">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_18">
    <item>
     <widget class="QLabel" name="sensorlabel">
      <property name="text">
       <string>Sensor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sensorEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
